libraries/AP_UAVCAN/AP_UAVCAN.cpp

```diff
@@ -258,6 +284,32 @@ void AP_UAVCAN::loop(void)
             continue;
         }
 
+        for (uint8_t i=0; i<INS_MAX_INSTANCES; i++) {
+            if (_imu_pub_mask & (1<<i)) {
+                if (_imu_accumulators[i].get_del_ang_dt() >= 1.0f/_imu_pub_frequency) {
+                    auto deltas = _imu_accumulators[i].pop_imu_deltas();
+                    Vector3f del_ang;
+                    deltas.del_quat.to_axis_angle(del_ang);
+
+                    org::ardupilot::sensor_output::IMUDeltas msg;
+                    msg.timestamp.usec = msg.timestamp.UNKNOWN;
```

Comment:
why have the timestamp if we're filling it with UNKNOWN?

Reply:
We will fill it in after we add UAVCAN time synchronization to ardupilot.